src/arrayMethodSort.js

```diff
@@ -4,9 +4,35 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareFunction2) {
```

Comment:
It looks strange with 2, not a good name. Better rename for example 👇 
```suggestion
  [].__proto__.sort2 = function(compareFunction = compare) {
```